modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -12,6 +12,8 @@ You can install the AWS Load Balancer Operator from the OperatorHub by using the
 
 * You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
 
+* Your cluster is configured with AWS set as the platform type and cloud provider.
```

Comment:
I think this is OK, but I read it wrong for second at first. I saw _set_ as a noun. Maybe s/set/configured ??

Reply:
@mburke5678 I think you're right, "set" is kind of redundant here since "configured" is already used at the beginning of the sentence, good eye!